docs/source/index.rst

```diff
@@ -57,6 +67,17 @@ On Ubuntu 14.04 (Trusty):
   :linenos:
   :lines: 20-
 
+System Requirements
+===================
+
+Although Cartographer may run on other systems, it is confirmed to be working
```

Comment:
Briefly looked into adding a CMake check. Doesn't look like it's straight forward or reliable. #46 calls it out as a known issue.